packages/odyssey-react/src/components/Select/Select/useChoices.tsx

```diff
@@ -93,7 +96,14 @@ const useChoices: UseChoices = ({
       }),
     });
 
-    return () => choices?.current?.destroy();
+    select.choices = choicesInstance;
+
+    return () => {
+      const select =
+        choicesInstance.passedElement as unknown as ChoicesHTMLSelectElement;
```

Comment:
I don't believe you need to redefine `select` here as the existing reference should be available from the closure unless there is some nuance here around targeting that DOM element during unmount that I'm not aware of?

Reply:
Seems quite obvious after you point it out 👍